lib/src/date-input/YearPicker.tsx

```diff
@@ -21,6 +22,8 @@ const YearPicker = ({ onYearSelect, selectedDate }: YearPickerPropsType): JSX.El
     yearToFocusEl?.focus();
```

Comment:
What do you think about changing the `scrollIntoView` parameters to:

`yearToFocusEl?.scrollIntoView?.({ block: "center", inline: "start" });`

so when the user opens the year picker the current focused year appears in the middle of the container and not at the bottom as it currently does.

Reply:
While this improves the scroll in the year picker this also makes the whole window scroll. I am not sure that I prefer this to how it was before.

Reply:
But it also happens with `block: "nearest"`. It's not related to the parameters, that's how `scrollIntoViewWorks`, takes the whole screen.

Reply:
Yes, the window scroll still happens with "nearest" but not every time the year picker is opened and the window is not exactly centered on the year to focus.

Reply:
Agree! For the moment, we stay like this.